app/models/nic/interface.rb

```diff
@@ -44,6 +44,22 @@ def vlanid
       return ''
     end
 
+    def mtu
```

Comment:
I know, you must probably hate me right now, but I'm not too fond of this approach. This is based on my believe, that a host can have a different mtu than the mtu that is configured on the subnet.
Let me give you an example of what is not possible by this:
Let's say, I use jumbo frames and have configured the subnet for an mtu of 9000 bytes. One of my hosts is not configured properly and only has a mtu of 1500 bytes configured. As I user I'd want to see this in foreman. E.g. the host should upload facts to foreman and foreman should display the current value on the host page. Even better would be to display a warning (e.g. by a host status).

What do you think about adding a field "mtu" to the nic model? We could do it in another PR to get the subnet changes in faster.

Reply:
Yes, I think you are right, we have to add this on per interface basis. I intended to add it in another PR, because I didn't have the time to dig into this. Do you want to add an mtu attribute to nic or get this from facts (or get it from facts and fallback to mtu attribute on nic ?) ? IMHO, I think that get it from facts and fallback to subnet value if not present is a good way to go.

I can add MTU to the show host's NICs tab too.

Reply:
> IMHO, I think that get it from facts and fallback to subnet value if not present is a good way to go.

I concur. I'd suggest: Let's remove this function (and dependent code) for now so we can get the first PR merged. Then let's add the code for adding mtu to an Interface in a separate PR. Deal?

Reply:
Never mind me, I was to slow. :-)